tests/jvm/src/test/resources/metac.expect

```diff
@@ -1065,9 +1065,8 @@ Occurrences:
 [42:21..42:25): head => scala/collection/IterableLike#head().
 
 Synthetics:
-[39:9..39:9):  => *[Unit]
-  [0:0..0:1): * => _star_.
-  [0:2..0:6): Unit => scala/Unit#
+[39:2..39:9): orig(locally) => orig(locally)[Unit]
```

Comment:
Also, as mentioned above, I don't think we should print `orig()` around the original tree on the lhs.